pkgs/by-name/av/avml/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  perl,
+  testers,
+  avml,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "avml";
+  version = "0.14.0";
+
+  src = fetchFromGitHub {
+    owner = "microsoft";
+    repo = pname;
+    rev = "v${version}";
```

It's better to be explicit.

```suggestion
    rev = "refs/tags/v${version}";
```